csrankings-s.csv

```diff
@@ -1279,6 +1279,7 @@ Simone Frintrop,University of Hamburg,https://www.inf.uni-hamburg.de/en/inst/ab/
 Simone L. Martins,UFF,http://www2.ic.uff.br/~simone,oEIT_KQAAAAJ
 Simone Linz,University of Auckland,https://unidirectory.auckland.ac.nz/profile/slin498,9lCabZsAAAAJ
 Simone Paolo Ponzetto,University of Mannheim,https://www.uni-mannheim.de/dws/people/professors/prof-dr-simone-paolo-ponzetto,VmIFG0EAAAAJ
+Simone Scalabrino,University of Molise,https://dibt.unimol.it/sscalabrino,FBP0Z64AAAAJ

```

Comment:
Web page says "research fellow"; only tenure-track full-time faculty are eligible for inclusion.